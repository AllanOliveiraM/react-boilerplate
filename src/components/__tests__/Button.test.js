import { shallow } from 'enzyme'
import React from 'react'
import renderer from 'react-test-renderer'
import Button from '../Button'

describe('<Button />', () => {
  it('renders a button element', () => {
    const component = renderer.create(<Button />)
    const tree = component.toJSON()
  })
})
