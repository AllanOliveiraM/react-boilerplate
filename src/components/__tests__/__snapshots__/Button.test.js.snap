// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> render and generate snapshot 1`] = `
<button
  className=""
  disabled={true}
/>
`;
